canu/utils/sls.py

```diff
@@ -87,59 +88,66 @@ def remote_request(
         # Get the admin client secret from Kubernetes
         #
         secret = None
-        try:
-            config.load_kube_config()
-            v1 = client.CoreV1Api()
-            secret_obj = v1.list_namespaced_secret(
-                "default",
-                field_selector="metadata.name=admin-client-auth",
-            )
-            secret_dict = secret_obj.to_dict()
-            secret_base64_str = secret_dict["items"][0]["data"]["client-secret"]
-            on_debug(
-                debug,
-                "base64 secret from Kubernetes is {}".format(secret_base64_str),
-            )
-            secret = base64.b64decode(secret_base64_str.encode("utf-8"))
-            on_debug(debug, "secret from Kubernetes is {}".format(secret))
-        except Exception as err:
-            print("Error collecting secret from Kubernetes: {}".format(err))
-            sys.exit(1)
+        cenv = os.getenv('CRAYENV')
+        csm = 'csm'
+        if cenv != csm:
```

Comment:
Best practice is to compare None with "is" or "is not" rather than "==" or "!=".

https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html
https://www.pythontutorial.net/advanced-python/python-none/